lldb/source/Target/Platform.cpp

```diff
@@ -1184,7 +1184,7 @@ bool Platform::IsCompatibleArchitecture(const ArchSpec &arch,
 Status Platform::PutFile(const FileSpec &source, const FileSpec &destination,
                          uint32_t uid, uint32_t gid) {
   Log *log = GetLog(LLDBLog::Platform);
-  LLDB_LOGF(log, "[PutFile] Using block by block transfer....\n");
+  LLDB_LOGF(log, "[PutFile] Using block by block transfer....");
```

Comment:
Why did this get changed?

Reply:
When testing this change, and using `log enable lldb platform` to read the logs, I just noticed this had a `\n` and removed it since `LLDB_LOGF` already adds a new line, and it didn't seem like most other logs had `\n`.

Is this bad to have in the patch?

Reply:
Since (I assume) you can't push directly, I suggest leaving this out of this PR and once this PR lands immediately put up a PR just to change that \n.

It's maybe a bit strict but PRs are free :)

Reply:
I wouldn't say it's bad to have it, but I wasn't sure why it was here. It seems unrelated to the intent of this PR, which is why I asked about it. I'm fine with you leaving it in this PR or submitting this in a different PR.

Reply:
I decided to leave it out of this PR